EIPS/eip-5748.md

```diff
@@ -0,0 +1,191 @@
+---
+eip: 5748
+title: Approval Expiration for EIP-20 Tokens
+description: This standard introduces a standardized way for users to set expiration limits on ERC-20 token approvals.
+author: Go+ Security (@GoPlusSecurity), Jeff Hall (@Mr-Lucky), Xavi (@XaaaaavitheFool), Turan Vural (@turanzv)
+discussions-to: <https://ethereum-magicians.org/t/eip-5748-automatic-expiration-of-token-approvals/>
+status: Draft
+type: Standards Track
+category: Interface
+created: 2022-10-02) format
+requires:
```

Comment:
This line is breaking the validator :rofl: I think you probably mean to depend on EIP-20 anyway:

```suggestion
requires: 20
```

Reply:
🙏🏼🙏🏼🙏🏼